### ISSUE TITLE
Disable Editing

### ISSUE THREAD

**Issue by harkor** [Frontwise/grid-editor#50]
Hello,

First, really nice work the best grid manager I ever seen for now :)

I would use this grid-editor only for edit grid and disable the edition of the text of the box.

You think it's possible ?

**Comment by Epskampie** [comment]
yes

**Comment by harkor** [comment]
Maybe say how ? :)

**Comment by Epskampie** [comment]
Maybe try it yourself first? ;-)

**Comment by harkor** [comment]
Thank you

**Comment by brouillon** [comment]
I do some test around that, and found a way to disallow some blocks to be edited with :
`/* Init RTE on click */
            canvas.on('click', '.ge-content', function(e) {
                if ($(this).parent().hasClass('no-rte') === false) {
                    var rte = getRTE($(this).data('ge-content-type'));
                    if (rte) {
                        rte.init(settings, $(this));
                    }
                }
            });`
So basically, if a col has class 'no-rte', it is not editable.

We can use a parameter to define a custom classname to stop editing some col and maybe one more to fully block editing ?

Do you plan to add this function to grid-editor ?
